maxim/tests/test_maxim_core_simple.py

```
@@ -0,0 +1,246 @@
+
+        new_cache = MaximInMemoryCache()
+        result = maxim.enable_prompt_management(cache=new_cache)
+
+        # Prompt management should be enabled
+        self.assertTrue(maxim.prompt_management)
+        # The new cache should be used
+        self.assertEqual(getattr(maxim, "_Maxim__cache"), new_cache)
+        # Should return self for method chaining
+        self.assertEqual(result, maxim)
+
+    def test_enable_prompt_management_starts_sync_thread(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_enable_prompt_management_starts_sync_thread`

Add return type annotation: `None`

(ANN201)